stratum/hal/config/x86-64-accton-wedge100bf-32qs-r0/chassis_config.pb.txt

```diff
@@ -373,7 +373,7 @@ singleton_ports {
   node: 1
 }
 singleton_ports {
-  id: 66
+  id: 33
```

Comment:
duplicated

Reply:
Good catch.

Should we remove the whole port? Everything but the name (`33/2`) is the same.

Reply:
the channel is different, these two ports are ports connected to the CPU, you can remove them(or one of them) if you like.